owncloudApp/src/main/java/com/owncloud/android/ui/activity/WhatsNewActivity.java

```diff
@@ -183,17 +189,17 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                                  @Nullable Bundle savedInstanceState) {
             View v = inflater.inflate(R.layout.whats_new_element, container, false);
```

Comment:
As this variable is only used for finding the UI elements and we won't need it anymore after using `binding_element`, it can be removed, do you agree?
BTW, this method returns this variable currently. Let's see if after removing it you can make out how to solve it 😉